database/migrations/2019_04_01_000000_drop_password_resets_and_stats_tables.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Migrations\Migration;
+
+class RemovePasswordResetsAndStatsTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::dropIfExists('password_resets');
+        Schema::dropIfExists('stats');
+    }
+}
```

Comment:
This migration needs a [corresponding `down()` function to reverse what happens in the `up()`](https://laravel.com/docs/5.6/migrations#migration-structure) so that it doesn't error out or get confused should we try to rollback any migrations. I think you should be able to grab exactly what is in the `up()` method of the original [`password_resets`](https://github.com/DoSomething/chompy/blob/master/database/migrations/2014_10_12_100000_create_password_resets_table.php#L16-L20) and migration, but for stats I think you'll have to combine [the original](https://github.com/DoSomething/chompy/blob/master/database/migrations/2018_04_19_144115_add_stats_table.php#L16-L21) and [another one that added a column](https://github.com/DoSomething/chompy/blob/master/database/migrations/2018_04_23_205823_add_filename_to_stats.php#L16-L18).

Reply:
Thanks, updated!